### ISSUE TITLE
Allow `textarea` `value` attribute in Svelte

### ISSUE THREAD

**Issue by kmoschcau** [markuplint/markuplint#1262]
- Markuplint version: `3.13.0`
- Parser lang: Svelte
- Node.js version: `18.17.1`
- OS: WSL2 on Windows 11

## Describe the bug

Normally the `value` attribute of a `textarea` shouldn't be used in markup. (Default values are given via its text content.) However Svelte does some magic here and allows binding to that attribute to set and read the `value` of a `textarea`. See [here](https://learn.svelte.dev/tutorial/textarea-inputs) for reference.

## Code Example

```html
<script>
  let description = "";
</script>

<textarea bind:value={description}></textarea>
```

## Steps To Reproduce

1. Run markuplint over the above code.

## The current behavior

Markuplint returns the following error:

> error: The "value" attribute is disallowed (invalid-attr)

## The expected behavior

Markuplint should not return an error.

**Comment by kmoschcau** [comment]
Thanks for the quick turnaround!
